staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/etcd.go

```diff
@@ -130,6 +130,19 @@ func (e *REST) List(ctx context.Context, options *metainternalversion.ListOption
 	return l, nil
 }
 
+func (r *REST) DeleteCollection(ctx context.Context, deleteValidation rest.ValidateObjectFunc, options *metav1.DeleteOptions, listOptions *metainternalversion.ListOptions) (runtime.Object, error) {
```

Comment:
Just two basic comments:
- Can we add comments about this method usages on top of this method. ?
- Along with adding new method can we also add the unit test which emits the expected behavior so that next person touching this method should get notified about the behavior and do not make mistakes.

Thanks

Reply:
+1 for a test :)

Reply:
@bhagwat070919 ping